src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -102,20 +105,20 @@ void LogConfiguration::post_initialize() {
 }
 
 void LogConfiguration::initialize(jlong vm_start_time) {
-  StdoutLog = new LogStdoutOutput();
-  StderrLog = new LogStderrOutput();
+  LogConfiguration::StdoutLog = new LogStdoutOutput();
+  LogConfiguration::StderrLog = new LogStderrOutput();
```

Comment:
Style nit: we're in a LogConfiguration method so we don't need to specify it again do we?